api/serializers.py

```diff
@@ -200,7 +200,7 @@ def to_representation(self, instance):
 class UserFeedbackSerializer(serializers.ModelSerializer):
     class Meta:
         model = UserFeedback
-        fields = ("reported_by", "message", "report_type", "created_at")
+        fields = ("reported_by", "message", "report_type", "created_at", "resolved")
         read_only_fields = ("reported_by",)
```

Comment:
Include the `resolved` field in the **read_only_fields** tuple as well and we're good to merge.